packages/icons/icons.json

```diff
@@ -2,3899 +2,3899 @@
+        "tags": "[analysis, investigation, search, study, test, brain, it\u2019s a brain, definitely a brain, could mean anything, but, yep, still a brain]",
         "group": "action",
         "codepoint": 62076
     },
     {
         "displayName": "Layers",
         "iconName": "layers",
-        "tags": "layers, levels, stack, cards",
+        "tags": "[layers, levels, stack, cards]",
         "group": "interface",
         "codepoint": 61979
     },
     {
         "displayName": "Locate",
         "iconName": "locate",
-        "tags": "target, location, destination, mark, map, area",
+        "tags": "[target, location, destination, mark, map, area, zoom, identify, locate, crosshairs]",
         "group": "action",
         "codepoint": 62001
     },
     {
         "displayName": "Bookmark",
         "iconName": "bookmark",
-        "tags": "bookmark, marker, holder, section, identifier, favorites",
+        "tags": "[bookmark, marker, holder, section, identifier, favorites, tag, save]",
         "group": "action",
         "codepoint": 61751
     },
     {
         "displayName": "Citation",
         "iconName": "citation",
-        "tags": "quotation, citation, marks, excerpt",
+        "tags": "[quotation, citation, marks, excerpt, quote, end quote, grammar, punctuation]",
         "group": "editor",
         "codepoint": 61780
     },
     {
         "displayName": "Tag",
         "iconName": "tag",
-        "tags": "tag, label, badge, identification",
+        "tags": "[tag, label, badge, identification]",
         "group": "action",
         "codepoint": 62175
     },
     {
         "displayName": "Clipboard",
         "iconName": "clipboard",
-        "tags": "clipboard, notepad, notebook, copy, paste, transfer, storage",
+        "tags": "[clipboard, notepad, notebook, copy, paste, transfer, storage, note]",
         "group": "action",
         "codepoint": 61783
     },
     {
         "displayName": "Selection",
         "iconName": "selection",
-        "tags": "selection, collection, circle, ring",
+        "tags": "[selection, collection, circle, ring]",
         "group": "action",
         "codepoint": 62117
     },
     {
         "displayName": "Events",
         "iconName": "timeline-events",
-        "tags": "calendar, timeframe, agenda, diary, day, week, month",
+        "tags": "[calendar, timeframe, agenda, diary, day, week, month, schedule]",
         "group": "interface",
         "codepoint": 62195
     },
     {
         "displayName": "Line chart",
         "iconName": "timeline-line-chart",
-        "tags": "graph, line, chart",
+        "tags": "[graph, line, chart, line chart, upward trend, time series, line graph]",
         "group": "data",
         "codepoint": 62196
     },
     {
         "displayName": "Bar chart",
         "iconName": "timeline-bar-chart",
-        "tags": "graph, bar, chart",
+        "tags": "[graph, bar, chart, horizontal histogram, bar chart]",
```

Comment:
Some refer to that as "horizontal" because the axis of the bases of the bars is horizontal. But on the other hand the bars themselves are vertical. I'll remove horizontal for simplicity--"histogram" should do.